tests/jerry/es2015/string-prototype-repeat.js

```diff
@@ -0,0 +1,73 @@
+/* Copyright JS Foundation and other contributors, http://js.foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+var z = "I am vengeance";
+try {
+    z.repeat(-1);
+    assert (false);
```

Comment:
For consistency, could you please use 2 spaces for alignment instead of 4? Also please use/do not use space before open parenthesis in the whole file.